### PR TITLE
Feature converteraudio

### PR DESCRIPTION
**Pull request by edsonarios**
- feature for converter audio
- feature for run command bash or cmd

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        String pathOutput = adaptedPath.substring(0, (adaptedPath.lastIndexOf("storage"))) + "output/";
+        String overwrite = " -y";
+        String command = ffmpeg + adaptedPath + bitrate + hz + volume + pathOutput + output + overwrite;
+        System.out.println(command);
+        runCommand.run(command);
+    }
+
+    /**
+     * Create parameter for volume
+     * @return
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: It should specify what is returning

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+     */
+    private String formatBitrate() {
+        if (!getBitrate().equals("")) {
+            return " -ab " + (Integer.parseInt(getBitrate()) * ONE_HUNDRED) + " ";
+        }
+        return "";
+    }
+
+    private String getFormat() {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:86]: Getters and Setters do not have a javadoc.

File: src/main/java/org/fundacion/jala/converter/service/RunCommand.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    /**
+     * Run command by cmd or bash
+     * @param command
+     */
+    public void run(final String command) {
+        ProcessBuilder processBuilder = new ProcessBuilder();
+
+        processBuilder.command("bash", "-c", command);
+
+        try {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/service/RunCommand.java:33]: A logger should be placed on this try and catch.

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java (hunk 1 above)
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: Maybe It should give more information about what returns

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    private String formatVolume() {
+        if (!getVolume().equals("") && !getVolume().equals("1")) {
+            return " -filter:a 'volume=" + getVolume() + "' ";
+        }
+        return "";
+    }
+
+    /**
+     * Create parameter for Hz
+     * @return
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: Maybe It should give more information about what returns

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+            if (getHz().equals("48")){
+                return " -ar 48000 ";
+            }
+        }
+        return "";
+    }
+
+    /**
+     * Create parameter for bitrate
+     * @return
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: Maybe It should give more information about what returns

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java (hunk 2 above)
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:86]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+            return " -ab " + (Integer.parseInt(getBitrate()) * ONE_HUNDRED) + " ";
+        }
+        return "";
+    }
+
+    private String getFormat() {
+        return format;
+    }
+
+    public void setFormat(final String newFormat) {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:94]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+        this.format = newFormat;
+    }
+
+    public String getBitrate() {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:101]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+        return bitrate;
+    }
+
+    public void setBitrate(final String newBitrate) {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:109]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+        this.bitrate = newBitrate;
+    }
+
+    public String getVolume() {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:116]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+        return volume;
+    }
+
+    public void setVolume(final String newVolume) {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:124]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+        this.volume = newVolume;
+    }
+
+    public String getHz() {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:131]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+        return hz;
+    }
+
+    public void setHz(final String newHz) {
```
Comment (JRojas999) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java:139]: It should have Javadoc

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+        this.hz = newHz;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: the methods must have their javadoc and specify what they return

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+        if (!getHz().equals("")) {
+            if (getHz().equals("20")){
+                return " -ar 22050 ";
+            }
+            if (getHz().equals("44")){
+                return " -ar 44100 ";
+            }
+            if (getHz().equals("48")){
+                return " -ar 48000 ";
+            }
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: the values 20 44 48 and their returns should be in constants

**Review by JRojas999 — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**